### PR TITLE
The veggydude patch 1

### PR DESCRIPTION
**Pull request by TheVeggydude**

### CODE REVIEW

**Comment by TWEApol** [comment]
What exactly does this do?

**Comment by TheVeggydude** [comment]
@TWEApol Changes made when the `master_pr` branch was active have not been merged back into `development`. I hope to prevent future merge conflicts with this PR.

File: pydash/pydash_app/dashboard/aggregator/__init__.py [hunk 1]
```diff
@@ -7,6 +7,10 @@
 from . import statistics
 
 
+def date_dict(dict):
```
Comment (TWEApol) [inline pydash/pydash_app/dashboard/aggregator/__init__.py]: This isn't used anywhere anymore.

**Comment by TWEApol** [comment]
@TheVeggydude In the future, could you notify me and let me decide for myself whether I deem the changes sufficient? (Github has this nice button for reviewers that have left a `requesting changes`-type of review :wink: )

**Pull request merged**